frontend/src/components/LevelMissionInfoBanner/LevelMissionInfoBanner.tsx

```diff
@@ -7,10 +7,16 @@ import './LevelMissionInfoBanner.css';
 function LevelMissionInfoBanner({
 	currentLevel,
 	openOverlay,
+	openLevelsCompleteOverlay,
+	numCompletedLevels,
 }: {
 	currentLevel: LEVEL_NAMES;
 	openOverlay: () => void;
+	openLevelsCompleteOverlay: () => void;
+	numCompletedLevels: number;
 }) {
+	const isLevelComplete = (currentLevel as number) < numCompletedLevels;
```

Comment:
you shouldn't need to cast here. LEVEL_NAMES is already a numeric type.

```suggestion
	const isLevelComplete = currentLevel < numCompletedLevels;
```

Reply:
I would even add
```typescript
const isLevel3 = currentLevel === LEVEL_NAMES.LEVEL_3
```
and incorporate that into the conditional render below, for brevity and explicitness in the render function

Reply:
yeah that makes sense